_maps/map_files/generic/CentCom.dmm

```diff
@@ -2307,10 +2307,7 @@
 /area/centcom/central_command_areas/briefing)
 "gG" = (
 /obj/effect/turf_decal/stripes/box,
-/obj/machinery/portable_atmospherics/pump{
-	desc = "A betrayer to pump-kind.";
-	name = "Lil Pump"
-	},
+/obj/machinery/portable_atmospherics/pump/lil_pump,
```

Comment:
Would you mind going back and adding the description to the one on the syndicate lavaland base?